src/dart-vm/index.md

```diff
@@ -33,6 +33,9 @@ You might find the following tutorials helpful.
 [Get Started](/tutorials/dart-vm/get-started)
 : Shows how to write a basic Dart script.
 
+[Dart gRPC Quickstart](https://grpc.io/docs/quickstart/dart.html)
+: Using the gRPC protocol for client/server communication.
```

Comment:
Let's make this parallel with the other descriptions by starting it with a third-person verb. Also, the gRPC docs call it a framework, not a protocol.

Walks you through running and modifying a client-server example that uses the gRPC framework.

Reply:
Done